scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -944,90 +946,94 @@
-                      matched-head? (handle-resource-offers! conn fenzo pool-name->pending-jobs-atom
-                                                             mesos-run-as-user @user->usage-future user->quota
-                                                             num-considerable offers
-                                                             rebalancer-reservation-atom pool-name compute-clusters)]
-                  (when (seq offers)
-                    (reset! resources-atom (view-incubating-offers fenzo)))
-                  ;; This check ensures that, although we value Fenzo's optimizations,
-                  ;; we also value Cook's sensibility of fairness when deciding which jobs
-                  ;; to schedule.  If Fenzo produces a set of matches that doesn't include
-                  ;; Cook's highest-priority job, on the next cycle, we give Fenzo it less
-                  ;; freedom in the form of fewer jobs to consider.
-                  (if matched-head?
-                    max-considerable
-                    (let [new-considerable (max 1 (long (* scaleback num-considerable)))] ;; With max=1000 and 1 iter/sec, this will take 88 seconds to reach 1
-                      (log/info "In" pool-name "pool, failed to match head, reducing number of considerable jobs"
-                                {:prev-considerable num-considerable
-                                 :new-considerable new-considerable
-                                 :pool pool-name})
-                      new-considerable)))
-                (catch Exception e
-                  (log/error e "In" pool-name "pool, offer handler encountered exception; continuing")
-                  max-considerable))]
-
-          (if (= next-considerable 1)
-            (counters/inc! iterations-at-fenzo-floor)
-            (counters/clear! iterations-at-fenzo-floor))
-
-          (if (>= (counters/value iterations-at-fenzo-floor) floor-iterations-before-warn)
-            (log/warn "In" pool-name "pool, offer handler has been showing Fenzo only 1 job for"
-                      (counters/value iterations-at-fenzo-floor) "iterations"))
-
-          (reset! fenzo-num-considerable-atom
-                  (if (>= (counters/value iterations-at-fenzo-floor) floor-iterations-before-reset)
-                    (do
-                      (log/error "In" pool-name "pool, FENZO CANNOT MATCH THE MOST IMPORTANT JOB."
-                                 "Fenzo has seen only 1 job for" (counters/value iterations-at-fenzo-floor)
-                                 "iterations, and still hasn't matched it.  Cook is now giving up and will "
-                                 "now give Fenzo" max-considerable "jobs to look at.")
-                      (meters/mark! fenzo-abandon-and-reset-meter)
-                      max-considerable)
-                    next-considerable))))
+        (log/info "Start match-jobs-event offer processing for " pool-name)
```

Comment:
```suggestion
        (log/info "In" pool-name "pool, starting offer matching")
```

(I think it'd be good to have "matching" in the log since we already have that in other matching-related log lines)